aya/src/sys/bpf.rs

```diff
@@ -151,7 +151,7 @@ pub(crate) fn bpf_load_program(
 
     if let Some(name) = &aya_attr.name {
         let name_bytes = name.to_bytes();
-        let len = cmp::min(name_bytes.len(), u.prog_name.len());
+        let len = cmp::min(name_bytes.len(), u.prog_name.len() - 1); // Ensure NULL termination.
```

Comment:
I don't think that TASK_COMM has to be NULL terminated?

Reply:
I tried that but got integration test failures :(